code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -285,11 +285,12 @@
 
 	data["misc"] += delta_time SECONDS * REM
 	M.adjust_jitter_up_to(4 SECONDS * delta_time, 20 SECONDS)
-	if(IS_CULTIST(M))
-		for(var/datum/action/innate/cult/blood_magic/BM in M.actions)
-			to_chat(M, span_cultlarge("Your blood rites falter as holy water scours your body!"))
-			for(var/datum/action/innate/cult/blood_spell/BS in BM.spells)
-				qdel(BS)
+	/// Commenting out Holy Water negating cult spells, will be changed functionality later
+	/* if(IS_CULTIST(M))
+		for(var/datum/action/innate/blood_cult/blood_magic/BM in M.actions) /// Removes the entire list of spells you can prepare as a cultist
+			/// to_chat(M, span_cultlarge("Your blood rites falter as holy water scours your body!"))
+			/// for(var/datum/action/innate/blood_cult/blood_spell/BS in BM.spells)
+				qdel(BS) */
```

Comment:
I know this is a WIP but don't forget to remove code instead of commenting it out

Reply:
oh yeah, I have a notepad specifically with all the comments that I can copy/paste once I'm ready to revisit them.